client/product_details/ThumbnailList.jsx

```diff
@@ -6,13 +6,18 @@ class ThumbnailList extends React.Component {
     super(props);
 
     this.state = {
-      firstPhotoIndex: 0
+      firstPhotoIndex: 0,
+      photoIndexList: null
     };
 
     this.handleUpScroll = this.handleUpScroll.bind(this);
     this.handleDownScroll = this.handleDownScroll.bind(this);
   }
 
+  componentDidMount() {
+    
+  }
+
```

Comment:
Not sure if this was left blank intentionally but just want to point out!

Reply:
Woops! I was going to try something different but then I gave what I had another shot and totally forgot about this. Will remove this as well!